mobile/app/build.gradle

```diff
@@ -60,7 +59,7 @@ android {
     buildTypes {
         debug {
             applicationIdSuffix ".debug"
-            versionNameSuffix "-test-$BUILD_DATE"
+            versionNameSuffix "-test-px-$BUILD_DATE"
```

Comment:
Apparently this is an unnecessary change

Reply:
Is a relict so i can install official version and this test side by side.. 

I change it back